lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -348,9 +349,6 @@ function _campaign_resource_reportback($nid, $values) {
     $values['file'] = dosomething_helpers_get_data_uri_from_fid($file->fid);
   }
 
-  // @todo: Move this logic into dosomething_reportback_save().
-  $rbid = dosomething_reportback_exists($nid, NULL, $uid);
```

Comment:
don't we need this check since we are looking for `rbid` in the if statement below?